src/components/NewSnippet.jsx

```diff
@@ -1,10 +1,60 @@
 import React from 'react';
+import { Scrollbars } from 'react-custom-scrollbars';
+import { Controlled as CodeMirror } from 'react-codemirror2';
 
+import 'codemirror/lib/codemirror.css';
+
+import Title from './common/Title';
+import Input from './common/Input';
+
+import '../styles/NewSnippet.styl';
 
 const NewSnippet = () => (
-  <div>
-    New Snippet page
-  </div>
+  [
+    <Title title="New snippet" key="title" />,
+    <div className="new-snippet" key="new-snippet">
+      <div className="new-snippet-code-wrapper">
+        <div className="new-snippet-code-header">
+          <Input placeholder="Title" />
+          <Input placeholder="Tags (separate tags by comma)" />
+        </div>
+        <div className="new-snippet-code">
+          <CodeMirror
+            value="console.log('All heil XSnippet')"
```

Comment:
nit: `hail`, otherwise it's in German ;)

Reply:
;) will fix